### DISCUSSION TITLE
regarding RDA analysis

### DISCUSSION THREAD

**Discussion by Sam131992** [ChiLiubio/microeco#226] [General]
Dear Microeco team,
When i perform RDA analysis for detecting the relationship between taxa and environmental parameters. I select evn table which contains 21 parameters but in the RDA analysis, only 5 parameters are visible. how can i include all the parameters in the figure?  
[RDA.pdf](https://github.com/ChiLiubio/microeco/files/11257717/RDA.pdf).
And Thank you sir for the help in GGally plot , it works.

**Comment by ChiLiubio** [comment]
Please write the issues in the "Issues part" (https://github.com/ChiLiubio/microeco/issues). Thanks.
Such issue need the reproduction of result. Please save and paste your data with the tutorial shows(https://chiliubio.github.io/microeco_tutorial/notes.html#save-function).
